src/Service/BookManager.php

```diff
@@ -83,38 +84,36 @@ public function updateBookLocation(Book $book, \SplFileInfo $file): Book
         return $book;
     }
 
+    public function createBookWithoutMetadata(\SplFileInfo $file): Book
+    {
+        $book = new Book();
+
+        $book->setTitle($file->getBasename());
+        $book->setChecksum($this->fileSystemManager->getFileChecksum($file));
+        $book->addAuthor('unknown');
+
+        $book->setExtension($file->getExtension());
+
+        $book->setBookPath('');
+        $book->setBookFilename('');
+
+        return $this->updateBookLocation($book, $file);
+    }
+
     /**
      * @return MetadataType
      *
      * @throws \Exception
      */
     public function extractEbookMetadata(\SplFileInfo $file): array
     {
-        try {
-            if (!Ebook::isValid($file->getRealPath())) {
-                throw new \RuntimeException('Could not read ebook'.$file->getRealPath());
-            }
+        if (!Ebook::isValid($file->getRealPath())) {
+            throw new BookExtractionException('Invalid eBook', $file->getRealPath());
+        }
```

Comment:
the Ebook::isValid throw errors sometimes, so I think it should remain in a trycatch